graphql/src/main/java/com/scalar/db/graphql/server/GraphQlServer.java

```diff
@@ -0,0 +1,63 @@
+package com.scalar.db.graphql.server;
+
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.service.StorageFactory;
+import java.io.FileInputStream;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import org.eclipse.jetty.server.Server;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "scalardb-graphql", description = "Starts Scalar DB GraphQL server.")
+public class GraphQlServer implements Callable<Integer> {
+  private static final Logger LOGGER = LoggerFactory.getLogger(GraphQlServer.class);
```

Comment:
It looks like it's never used?

Reply:
I was planning to add LOGGERs to the GraphQL components since they do not log properly. I will work on this in another task, so I removed this line for now.